bin/main.rb

```diff
@@ -1,80 +1,62 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
-class Test
-  attr_accessor :board, :input, :x, :o
+class Game
+  require_relative '../lib/board'
+  require_relative '../lib/players'
 
-  def initialize
-    @x = x
-    @o = o
-    @input = input
-    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
-  end
+  attr_accessor :curr_input, :curr_player
 
-  def display_board
-    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
-    puts ' ----------- '
-    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
-    puts ' ----------- '
-    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
```

Comment:
- [ ] All game-logic related resources (classes and/or modules) are kept in the `lib` directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))